frontend/src/pages/org/collections-list.ts

```diff
@@ -437,7 +416,8 @@ export class CollectionsList extends BtrixElement {
       `;
     }
 
-    const searchResults = this.fuse.search(this.searchByValue).slice(0, 10);
+    const searchResults =
+      this.searchOrg.collections?.search(this.searchByValue).slice(0, 10) || [];
```

Comment:
Similar here, this could use fuse's built-in `limit` option
```suggestion
      this.searchOrg.collections?.search(this.searchByValue, { limit: 10 }) ||
      [];
```